lib/beaker/tasks/quick_start.rb

```diff
@@ -1,49 +1,93 @@
 require 'beaker-hostgenerator'
 
+VAGRANT  = ['ubuntu1404-64default.mdcal-ubuntu1404-64af', '--hypervisor=vagrant',
+            '--global-config=box_url=https://vagrantcloud.com/puppetlabs/boxes/ubuntu-14.04-64-nocm,box=puppetlabs/ubuntu-14.04-64-nocm']
```

Comment:
@nwolfe Yeah both ways seem to work in code, from the command line though if i use the {} it only generates the last key value pair which is why I held off from doing this in code. I've updated it though as it makes sense to do it right.

FYI from the command line:

beaker-hostgenerator ubuntu1404-64default.mdcal-ubuntu1404-64af --hypervisor=vagrant --global-config={box_url=https://vagrantcloud.com/puppetlabs/boxes/ubuntu-14.04-64-nocm,box=puppetlabs/ubuntu-14.04-64-nocm}

produces:
CONFIG:
  nfs_server: none
  consoleport: 443
  box: puppetlabs/ubuntu-14.04-64-nocm

If I run: 

beaker-hostgenerator ubuntu1404-64default.mdcal-ubuntu1404-64af --hypervisor=vagrant --global-config=box_url=https://vagrantcloud.com/puppetlabs/boxes/ubuntu-14.04-64-nocm,box=puppetlabs/ubuntu-14.04-64-nocm

I get as expected:

CONFIG:
  nfs_server: none
  consoleport: 443
  box_url: https://vagrantcloud.com/puppetlabs/boxes/ubuntu-14.04-64-nocm
  box: puppetlabs/ubuntu-14.04-64-nocm